app/config/configuration.py

```diff
@@ -0,0 +1,17 @@
+import os
+from functools import lru_cache
+from pathlib import Path
+
+from pydantic import BaseSettings
+
+
+class Settings(BaseSettings):
+    DB_FILE_NAME: str = "Alinka.db"
+    BASE_PATH: Path = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
```

Comment:
output is `str`, not `Path`:
```suggestion
    BASE_PATH: str = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
```